hepdata/modules/email/api.py

```diff
@@ -310,6 +311,43 @@ def send_cookie_email(submission_participant,
                            message_body,
                            reply_to_address=coordinator.email)
 
+def send_reminder_email(submission_participant, record_information, version, show_detail=False):
+
+    site_url = current_app.config.get('SITE_URL', 'https://www.hepdata.net')
+    tables = []
+    hepsubmission = get_latest_hepsubmission(
+        publication_recid=record_information['recid']
+    )
+    coordinator = User.query.get(hepsubmission.coordinator)
+
+    reviewers_notified = submission_participant.user_account is not None
+
+    message_body = render_template('hepdata_theme/email/submission_status.html',
+                                   name=submission_participant.full_name,
+                                   actor=coordinator.email,
+                                   article=record_information['recid'],
+                                   message=None,
+                                   invite_token=submission_participant.invitation_cookie,
+                                   role=submission_participant.role,
+                                   show_detail=show_detail,
+                                   data_tables=tables,
+                                   reviewers_notified=reviewers_notified,
+                                   title=record_information['title'],
+                                   site_url=site_url,
+                                   reminder=True,
+                                   link=site_url + "/record/{0}"
+                                   .format(record_information['recid']))
+
+    if not reviewers_notified:
+        message_subject = '[HEPData] Reminder to review submission {0}'.format(record_information['recid'])
+    else:
+        message_subject = '[HEPData] Reminder about submission {0}'.format(record_information['recid'])
```

Comment:
Instead of using `reviewer_notified` in these conditions I would check for `if submission_participant.role == 'reviewer'` to use the first subject, `else` use the second subject.